src/actions/ratingsActions.js

```diff
@@ -0,0 +1,27 @@
+import { RATE_ACCOMMODATION, RATE_ACCOMMODATION_ERROR, SHOW_ALERT } from './types';
+import backendCall from '../helpers/backendCall';
+import { getToken } from '../helpers/authHelper';
+
+const ratingsType = (type, payload) => ({
+  type,
+  payload,
+});
+
+const token = getToken();
+
+const headers = {
+  Authorization: `Bearer ${token}`,
+};
+
+export const rateAccommodation = (ratings, setShow) => async (dispatch) => {
+  try {
+    const res = await backendCall.post('/ratings', ratings, { headers });
```

Comment:
Add a new line

Reply:
My editor will not allow me to do so. It reverts due to the auto-fix on save